docs/PLUGIN_DEVELOPMENT.md

```
@@ -326,6 +328,134 @@ public void onBlockPlace(PlayerBlockPlaceEvent event) {
+@Subscribe
+public void onPlayerTransfer(PlayerTransferEvent event) {
+    if (isServerUnderMaintenance(event.getTargetServer().getName())) {
+        event.setResult(PlayerTransferEvent.PlayerTransferResult.denied());
+    }
+}
+```
+
+**API Reference:**
+
+| `PlayerTransferEvent` Method | Description |
+|------------------------------|-------------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)